modules/reporting/app/helpers/reporting_helper.rb

```diff
@@ -142,7 +142,7 @@ def field_sort_map(key, value)
     case key.to_sym
     when :work_package_id, :tweek, :tmonth, :week  then value.to_i
     when :spent_on                                 then value.to_date.mjd
-    else h(field_representation_map(key, value).gsub(/<\/?[^>]*>/, ''))
+    else h(Sanitize.fragment(field_representation_map(key, value)))
```

Comment:
I hesitated to use strip_tags. The reason I did not use it is because I was not sure the helper was available, and I am always a bit confused about which module should be included.
I looked into the `strip_tags` implementation and saw that it was using Loofah and nokogiri under the hood, so I used Sanitize instead which does the same.

I'll switch to `strip_tags` and see if that makes brakeman happy.

Reply:
Done.